### PR TITLE
Remove icons, add post install command

### PR DESCRIPTION
**Pull request by viktor-yakubiv**

### CODE REVIEW

**Comment by viktor-yakubiv** [comment]
@Joozty I do not know how to test this properly, `npm link` does not help me with this.

The idea is that `npx @oacore/design build` is called after the package installation via `postinstall` script. The command builds the sprite into the directory specified by `design.config.js` and creates a config file with the public sprite location, e.g. `/assets/icons.svg` that can be used by components later.

I do not know how to call postinstall without publishing 🤔

**Comment by Joozty** [comment]
Well, I think you can go with the method trial and error here, i.e. merge-release-repeat.

**Comment by viktor-yakubiv** [comment]
I installed it to an empty project with `design.config.js` from GitHub with the commit hash specified. It works exactly as expected.

If you need more details, check the snippet below.

#### `package.json`
```json
{
  "dependencies": {
    "@mdi/svg": "^5.1.45",
    "@oacore/design": "github:oacore/design#e86aaafcaf1f676527259e582ae26cea24338a52"
  }
}
```

#### `design.config.js`
```js
const path = require('path')

const icons = [
  'alert-circle',
]

const iconsRoot = path.join(
  path.dirname(require.resolve('@mdi/svg/package.json')),
  './svg'
)

const config = {
  icons: {
    path: iconsRoot,
    files: icons,
  },

  output: {
    path: path.join(__dirname, 'public'),
    publicPath: '/',
    icons: {
      files: 'icons',
      sprite: 'icons.svg',
    },
  },
}

module.exports = config
```

File: src/elements/icon/icon.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ example.
 
 ```jsx
 <button type="button">
-  <Icon src="/icons.svg#cite" alt="Cite" aria-hidden />
-  Cite
+  <Icon src="chech.svg" aria-hidden />
```
Comment (Joozty) [inline src/elements/icon/icon.md:21]:
```suggestion
  <Icon src="check.svg" aria-hidden />
```

**Pull request merged**